robot/rospackages/src/task_handler/scripts/task_handler_client.py

```diff
@@ -40,24 +50,40 @@ def usage():
     """
     Return string showcasing proper usage of this client script
     """
-    help_msg = """USAGE:\nrosrun task_handler task_handler_client.py [task] [status] optional:[args]
-                  \nValid task options: ['arm_listener', 'arm_listener', 'science_listener', 'camera_stream']
-                  \nValid status options: [0, 1, 2]
-                  \nValid camera stream args: ['/dev/ttyFrontCam', '/dev/ttyRearCam', '/dev/ttyArmScienceCam']"""
+    global ports
+
+    help_msg = "USAGE:\nrosrun task_handler task_handler_client.py [task] [status] optional:[args]"
+    help_msg += "\nValid task options: ['arm_listener', 'arm_listener', 'science_listener', 'camera_stream']"
+    help_msg  += "\nValid status options: [0, 1, 2]"
+
+    if local:
+        help_msg +="\nValid camera stream args: ["
+        help_msg += "'" + "' '".join(ports) + "']"
+    else:
+        help_msg  += "\nValid camera stream args: ['/dev/ttyFrontCam', '/dev/ttyRearCam', '/dev/ttyArmScienceCam']"
+
     return help_msg
 
 if __name__ == "__main__":
     # check for invalid ranges for args
-    if len(sys.argv) < 3 or len(sys.argv) > 4:
+    if len(sys.argv) < 3 or len(sys.argv) > 5:
         print(usage())
         sys.exit(1)
 
+    local = False
+    ports = []
+
+    if "local" in sys.argv:
+        print("supported options:")
+        print("- camera_stream")
+        local = True
+
     rospy.init_node("task_handler_client")
 
     task = sys.argv[1]
     status = int(sys.argv[2])
 
-    if len(sys.argv) == 4:
+    if len(sys.argv) >= 4:
         args = sys.argv[3]
     else:
         args = ""
```

Comment:
what is this for

Reply:
I think it was meant to cause an error to be thrown by the following `is_valid_request()` check.
I have removed the `else` block since the effect is the same.